flink-connector-oracle-cdc/src/main/java/com/ververica/cdc/connectors/oracle/table/OracleTableSourceFactory.java

```diff
@@ -54,6 +54,13 @@ public class OracleTableSourceFactory implements DynamicTableSourceFactory {
                     .defaultValue(1521)
                     .withDescription("Integer port number of the Oracle database server.");
 
+    private static final ConfigOption<String> URL =
```

Comment:
Could you also add the optional config to the connector document？

Reply:
yes